MapofAccountNameAndId.cls

```diff
@@ -0,0 +1,38 @@
+public with sharing class MapofAccountNameAndId {
```

Comment:
Is the class name justifying the actual requirement or problem statement.
Is the problem statement followed?